site/_docs/reference.md

```diff
@@ -2730,6 +2730,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | p | DIFFERENCE(string, string)                     | Returns a measure of the similarity of two strings, namely the number of character positions that their `SOUNDEX` values have in common: 4 if the `SOUNDEX` values are same and 0 if the `SOUNDEX` values are totally different
 | b | ENDS_WITH(string1, string2)                    | Returns whether *string2* is a suffix of *string1*
 | o | EXTRACT(xml, xpath, [, namespaces ])           | Returns the XML fragment of the element or elements matched by the XPath expression. The optional namespace value that specifies a default mapping or namespace mapping for prefixes, which is used when evaluating the XPath expression
+| s | EXISTS(array, predicate)                       | Returns whether a *predicate* holds for one or more elements in the *array*
```

Comment:
you should describe what the `predicate` argument means. Not in this row, but before the table, where other arguments are described.